org.Bhupesh-V.ugit.yml

```diff
@@ -0,0 +1,45 @@
+app-id: org.Bhupesh-V.ugit
```

Comment:
this app id doesn't meet the requirements.

1. `-` are not allowed anywhere but the last component.
2. the domain doesn't exist.